### PR TITLE
Improve cen_bandwidth_package_attachment resource use common method.

### PR DESCRIPTION
**Pull request by Lexsss**

### CODE REVIEW

File: alicloud/resource_alicloud_cen_bandwidth_package_attachment.go [hunk 1]
```diff
@@ -51,18 +50,18 @@ func resourceAlicloudCenBandwidthPackageAttachmentCreate(d *schema.ResourceData,
 		})
 		if err != nil {
 			if IsExceptedErrors(err, []string{InvalidBwpInstanceStatus, InvalidBwpBusinessStatus, InvalidCenInstanceStatus}) {
-				return resource.RetryableError(fmt.Errorf("Associate bandwidth package %s to CEN %s timeout and got an error: %#v", cenBwpId, cenId, err))
+				return resource.RetryableError(err)
 			}
 			return resource.NonRetryableError(err)
 		}
```
Comment (YY-tomorrow) [inline alicloud/resource_alicloud_cen_bandwidth_package_attachment.go]: add addDebug()

File: alicloud/resource_alicloud_cen_bandwidth_package_attachment.go [hunk 2]
```diff
@@ -100,27 +99,23 @@ func resourceAlicloudCenBandwidthPackageAttachmentDelete(d *schema.ResourceData,
...
-		return fmt.Errorf("Unassociate bandwidth %s from CEN %s got an error: %#v.", cenBwpId, cenId, err)
+		return WrapErrorf(err, DefaultErrorMsg, d.Id(), request.GetActionName(), AlibabaCloudSdkGoERROR)
 	}
 
-	if err := cenService.WaitForCenBandwidthPackageAttachment(cenBwpId, Idle, DefaultCenTimeout); err != nil {
-		return fmt.Errorf("Timeout when WaitForCenBandwidthPackageAttachment, CEN ID %s, bandwidth package ID %s, error %#v", cenId, cenBwpId, err)
-	}
-
-	return nil
+	return cenService.WaitForCenBandwidthPackageAttachment(cenBwpId, Idle, DefaultCenTimeout)
```
Comment (YY-tomorrow) [inline alicloud/resource_alicloud_cen_bandwidth_package_attachment.go]: please add WrapError()

File: alicloud/service_alicloud_cen.go [hunk 3]
```diff
@@ -232,40 +232,43 @@ func (s *CenService) WaitForCenBandwidthPackageUpdate(cenBwpId string, bandwidth
...
+		object, err := s.DescribeCenBandwidthPackage(id)
 		if err != nil {
-			return err
+			if !NotFoundError(err) {
+				if status == Idle {
+					return nil
+				}
+			} else {
+				return WrapError(err)
+			}
```
Comment (YY-tomorrow) [inline alicloud/service_alicloud_cen.go]: what is this

File: alicloud/resource_alicloud_cen_bandwidth_package_attachment_test.go [hunk 4]
```diff
@@ -4,98 +4,85 @@ import (
...
+
+	testAccCheck := rac.resourceAttrMapUpdateSet()
+	rand := acctest.RandIntRange(1000000, 9999999)
+	name := fmt.Sprintf(`"tf-testAcc%sCenBandwidthPackageAttachment-%d"`, defaultRegionToTest, rand)
+	testAccConfig := resourceTestAccConfigFunc(resourceId, name, resourceCenBandwidthPackageAttachmentConfigDependence)
+
 	resource.Test(t, resource.TestCase{
 		PreCheck: func() {
 			testAccPreCheck(t)
-			testAccPreCheckWithAccountSiteType(t, DomesticSite)
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_cen_bandwidth_package_attachment_test.go]: why remove the line 19?

**Pull request merged**